Combobox.js

```diff
@@ -960,8 +1000,10 @@ define([
 		_dropDownKeyUpHandler: dcl.superCall(function (sup) {
 			return function () {
 				if (this.hasDownArrow) {
-					if (this.inputNode.value.length === 0 && this.minFilterChars === 0) {
-						this.list.query = this._getDefaultQuery();
+					if (this.inputNode.value.length === 0) {
+						if (this.minFilterChars === 0) {
+							this.list.query = this._getDefaultQuery();
+						}
```

Comment:
I guess this is the code change to "Restore down-arrow key original behaviour, even if the inputNode is empty."?

It doesn't seem quite right.  For example:

* What if the list hasn't been created yet?  Didn't we agree to always set the list's query at the same time as we create the list?
* What if the user typed a different key than the down arrow, such as backspace?